azure/apis/database/v1alpha1/sql_types.go

```diff
@@ -33,6 +34,8 @@ const (
 	OperationCreateFirewallRules = "createFirewallRules"
 )
 
+// +kubebuilder:object:generate=false
```

Comment:
is there a reason for `generate=false`?

Reply:
Yes. Without the tag it will attempt to generate `DeepCopy` functions for the `SQLServer` interface, which we do not want.